hermes-common/src/main/java/pl/allegro/tech/hermes/common/di/CommonBinder.java

```diff
@@ -8,24 +8,7 @@
 import pl.allegro.tech.hermes.common.broker.ZookeeperBrokerStorage;
 import pl.allegro.tech.hermes.common.clock.ClockFactory;
 import pl.allegro.tech.hermes.common.config.ConfigFactory;
-import pl.allegro.tech.hermes.common.di.factories.BoonObjectMapperFactory;
-import pl.allegro.tech.hermes.common.di.factories.CuratorClientFactory;
-import pl.allegro.tech.hermes.common.di.factories.DistributedEphemeralCounterFactory;
-import pl.allegro.tech.hermes.common.di.factories.GraphiteWebTargetFactory;
-import pl.allegro.tech.hermes.common.di.factories.GroupRepositoryFactory;
-import pl.allegro.tech.hermes.common.di.factories.HermesCuratorClientFactory;
-import pl.allegro.tech.hermes.common.di.factories.KafkaCuratorClientFactory;
-import pl.allegro.tech.hermes.common.di.factories.MessagePreviewRepositoryFactory;
-import pl.allegro.tech.hermes.common.di.factories.MetricRegistryFactory;
-import pl.allegro.tech.hermes.common.di.factories.ModelAwareZookeeperNotifyingCacheFactory;
-import pl.allegro.tech.hermes.common.di.factories.ObjectMapperFactory;
-import pl.allegro.tech.hermes.common.di.factories.PathsCompilerFactory;
-import pl.allegro.tech.hermes.common.di.factories.SharedCounterFactory;
-import pl.allegro.tech.hermes.common.di.factories.SimpleConsumerPoolFactory;
-import pl.allegro.tech.hermes.common.di.factories.SubscriptionOffsetChangeIndicatorFactory;
-import pl.allegro.tech.hermes.common.di.factories.SubscriptionRepositoryFactory;
-import pl.allegro.tech.hermes.common.di.factories.TopicRepositoryFactory;
-import pl.allegro.tech.hermes.common.di.factories.ZookeeperPathsFactory;
+import pl.allegro.tech.hermes.common.di.factories.*;
```

Comment:
Well, as long as wildcard imports don't impact runtime performance I'm not against them. :> Anyway, I've changed my IDE setting and reverted the code (in all other classes as well).